cli/src/cmd_net.rs

```diff
@@ -311,7 +319,13 @@ impl AuthenticatedCmd for CmdBgpWithdraw {
             })
             .collect();
 
+        let before = current.len();
         current.retain(|x| x.network.to_string() != self.prefix.to_string());
+        let after = current.len();
+        if before == after {
+            eprintln_nopipe!("no originated prefixes match the provided args");
+            return Ok(());
```

Comment:
Do we want to return an anyhow error or does that not produce the desired result? Do we test this (do we want to)?